### DISCUSSION TITLE
Can a theme be set via the command line when starting btop?

### DISCUSSION THREAD

**Discussion by gm1168** [aristocratos/btop#777] [Q&A]
Looking at the man page for btop, it looks like there are no command line switches to allow a theme to be set (vs. what is in the btop.conf file).  Is there a command line option for setting the theme, or would that require an enhancement request?
Thank you.

**Comment by HealthyPear** [comment]
you can press the 'm' key to enter the options and change it from there so you can see in real time how it looks

**Comment by ShreeChaturvedi** [comment]
Copy `~/.config/btop/btop.conf` to `~/.config/btop/night.conf` and set:

```
color_theme = "nord"
```

Then start with `btop -c ~/.config/btop/night.conf`. Don't strip the copy down to that one line. A file with only `color_theme` uses stock defaults for every other key. The name is the stem of a `.theme` file (`nord`, `gruvbox_dark`, `Default`) under `~/.config/btop/themes` or the packaged themes dir. `nord` is already in the tree. Same line in your usual `btop.conf` if you want it every launch.
